driver/pool_driver.hpp

```diff
@@ -455,25 +455,27 @@ int PoolDriver_impl<Tgpu, Tref, Index>::RunForwardGPU()
 
     Timer t;
     START_TIME
+    int rc = 0;
```

Comment:
```suggestion
    static_assert(miopenStatusSuccess == 0);
    int rc = miopenStatusSuccess;
```
Otherwise bitwise OR logic is broken.

Reply:
`miopenStatusSuccess == 0` is carved in stone (in the public API) and should never change. Let me keep 0 for the sake of the old-style hackery.